luigi/worker.py

```diff
@@ -238,8 +238,11 @@ class DequeQueue(collections.deque):
     deque wrapper implementing the Queue interface.
     """
 
-    put = collections.deque.append
-    get = collections.deque.pop
+    def put(self, *args, **kwargs):
```

Comment:
The one instance I can find of passing in any kwargs is worker.py line 746.  So I presume DequeQueue needs to handle timeouts?  Anyone want to volunteer for that?  This is preventing me from using luigi in Amazon Lambda.